__tests__/app.test.js

```diff
@@ -32,4 +32,38 @@ describe('1 GET/api/topics', () => {
             expect(body.msg).toBe('route does not exist')
         })
     });
+});
+describe('2. GET API ARTICLES', () => {
+    test('status 200, should respond with an articles object with an array of articles', () => {
+        return request(app).get("/api/articles").expect(200).then((response)=>{
+            expect(response.body.articles).toEqual(expect.any(Array))
+            expect(response.body.articles.length).not.toEqual(0)
```

Comment:
You could assert more than it being greater than 0 and check it is the expected length.